.github/workflows/test_adapters.yaml

```diff
@@ -150,7 +150,16 @@ jobs:
       - name: Pattern apply (Install service mesh)
         run: |
           cat ./${{ inputs.patternfile_name }}
-          mesheryctl pattern apply -f ./${{ inputs.patternfile_name }}
+          for n in {1..3}
+          do
+            x=$(mesheryctl pattern apply -f ./${{ inputs.patternfile_name }})
+            if [[ $x == "*invalid workload of type:*" ]];
```

Comment:
@Revolyssup @Zanda256  Is this the only error output that could be produced? 

Besides, Issue #6102 states the reason for re-trying is "will fail to deploy a sample pattern **in the event that Meshery Server has yet to connect to Kubernetes**" so `invalid workload of type:` doesn't feel like something mesheryctl would throw back at a user in such case **(remember our CI pipeline spins up everything locally so we're trying to address a scenario that an end user might not experience too frequently locally but our CI pipeline like ours would)**